docs/API/panel/table/index.md

```diff
@@ -231,7 +231,7 @@ fieldConfig.defaults.custom.withCellOptionsMixin(value)
 
 PARAMETERS:
 
-* **value** (`string`)
+* **value** (`object`)
```

Comment:
If this is supposed to be no-op, how come we have a meaningful change here?

Reply:
This is the result of a fix in crdsonnet, it now recursively looks for the correct `type`, before if it didn't find a `type` attribute it assumed it was a string, now it looks further and defaulting to a string as a real last resort. Related PR: https://github.com/crdsonnet/crdsonnet/pull/16/

Reply:
:+1: Okay, so edit the description then I can approve :-)